docs/deploying-cf-aws.MD

```diff
@@ -0,0 +1,374 @@
+```
+
+You should see something like this:
+
+```bash
+$> eval "$(bbl print-env)"; bosh log-in
+Successfully authenticated with UAA
+
+Succeeded
+```
+
+## Deploy Cloud Foundry in AWS
+We're half way through, at this point there should be 2 EC2 instances in AWS that are the jumpbox and the bosh director. The `vars/` subdirectory contains the terraform and other sensitive files to connect to the bosh director as well as to authenticate through the jumpbox.
+
+### Clone the cf-deployment repository:
+Before installing Cloud Foundry, you will need to clone the cf-deployment repository, which contains the configuration to setup Cloud Foundry in a development environment:
+
+```
+git clone https://github.com/cloudfoundry/cf-deployment.git
+```
+
+### Validate OS Stemcell based on cf-deployment
+
+The OS stemcell defined in bbl might not match the one in the cf-deployment.yaml. During the bosh director deployment, bbl will upload its preconfigured version, but it might not match the one defined in the `cf-deployment/cf-deployment.yml` file. Check the output from the `bbl up` command and locate the line where it mentions which stemcell version is uploading:
+
+```bash
+...
+Finished installing CPI (00:03:30)
+
+Uploading stemcell 'bosh-aws-xen-hvm-ubuntu-jammy-go_agent/1.822'... Finished (00:04:23)
+
+Started deploying
+...
+```
+
+Compare it with the one in the `cf-deployment/cf-deployment.yml` file:
+
+```
+tail -n4 cf-deployment/cf-deployment.yml
+```
+
+Example:
+```
+$> tail -n4 cf-deployment/cf-deployment.yml
+stemcells:
+- alias: default
+  os: ubuntu-jammy
+  version: "1.824"
+```
+
+If they don't match, upload the stemcell specified in the cf-deployment.yml. The following command is also retrieveable from the bosh.io website https://bosh.io/stemcells/bosh-aws-xen-hvm-ubuntu-jammy-go_agent )
+```
+bosh upload-stemcell --sha1 c040c8101a39f3118b4b2155262d10bb4b9c2d57 "https://bosh.io/d/stemcells/bosh-aws-xen-hvm-ubuntu-jammy-go_agent?v=1.824"
+
+```
+
+You can also override the one in the `cf-deployment.yml` or specify the version in `bbl plan`, whichever solution works better for you.
+
+## Deploy CF
+At this point, we're ready to deploy Cloud Foundry. Run the following command to start the deployment:
+
+```bash
+bosh -d cf deploy cf-deployment/cf-deployment.yml \
+-o cf-deployment/operations/aws.yml \
+-o cf-deployment/operations/use-latest-stemcell.yml \
+-o cf-deployment/operations/scale-to-one-az.yml \
+-v system_domain=$SYSTEM_DOMAIN \
+-n
+```
+
+This command will take roughly 90 minutes to complete.
+
+### Configure your local environment to be able to communicate with the Cloud Foundry instance
+You will need to configure your `/etc/hosts` to be able to communicate with the Cloud Foundry services using the system domain we provided. First we'll retrieve the router LB with the `aws` cli and then we'll resolve its domain name to one of its IPs so we can add an entry in the /etc/hosts with the Cloud Foundry subdomains:
```

Comment:
> First we'll retrieve the router LB
Can add the acronym definition like `Load Balancer (LB)` to ease the understanding?

Reply:
Done.